libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -356,7 +356,7 @@ const AP_Param::GroupInfo AP_GPS::var_info[] = {
 #if AP_GPS_SBF_ENABLED
     // @Param: _COM_PORT
     // @DisplayName: GPS physical COM port
-    // @Description: The physical COM port on the connected device, currently only applies to SBF GPS
+    // @Description: The physical COM port on the connected device, currently only applies to SBF and GSOF GPS
```

Comment:
it should explain that 0 == COM1 and 1 == COM2
alternatively, you could make it be 1 == COM1 and 2 == COM2 which is less confusing for users, and do a set_default(2) within the GSF driver so it defaults to COM2 for GSF users

Reply:
What do you think about the wiki entry I made that explains it?

Reply:
Here is a screenshot of all the available ports. Only two of them are called "COM". I can ask Trimble tomorrow whether the other ports are exposed in a similar way. 
![image](https://github.com/ArduPilot/ardupilot/assets/25047695/3c2449ab-7e2a-4f15-93dd-75df7b186621)

Reply:
it could be done as "@Values: 0:COM1, 1:COM2, 2:COM3"

Reply:
Done. Added a set_default call in the constructor of the GSOF driver, and added the `@values` specifier.